READme.md

```diff
@@ -37,6 +103,35 @@ Omaidu Othniel
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [coming soon!!] **[coming_soon]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+coming soon!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+coming soon!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
You did well in adding back all sections as requested by the previous reviewer, however, kindly tailor the details in all sections to be in alignment with your project 😄 .. for instance, the `setup` section doesn't include this repo in the details, please add tools to be installed to run the project successfully, add future features and acknowledge people who made this project successful.